cmd/vaults/commands.go

```diff
@@ -687,19 +732,28 @@ func newWalletCommand() *cli.Command {
 			{
 				Name:      "address",
 				Usage:     "Print the public key for an account's private key",
-				UsageText: "vaults account address <file_path>",
-				Description: "The result of the `vaults account create` command will write a private key to a file, \n" +
-					"and this lets you retrieve the public key value for use in other commands.\n\n" +
-					"EXAMPLE:\n\nvaults account address /path/to/file",
+				UsageText: "vaults account address <file_path|hex_string>",
```

Comment:
this is a backlog issue from the UX improvements project: [ENG-787](https://linear.app/tableland/issue/ENG-787/allow-vaults-account-address-to-take-a-string)

i.e., you can do `vaults account address /path/to/pk-file`, or you can do `vaults account address abcd1234`. i thought about adding a boolean flag like `--type file` or `--type string` in case that's a better UX, but it's a small feature

Reply:
yeah, i also prefer being explicit by using flags, but looks good